tests/themis/themis_secure_cell.c

```diff
@@ -26,7 +26,7 @@
 #define MAX_MESSAGE_SIZE 4096
 
 /* Keep it under 2^31 to support 32-bit systems. */
-#define CORRUPTED_LENGTH 0x1337BEEF
```

Comment:
an era has gone :(

Reply:
It just occured to me that simply `0xBEEF`, or `0xC0DE`, or maybe even `0x31337` should be fine. Got any favorites?